addons/mass_mailing/wizard/mailing_contact_import.py

```diff
@@ -15,6 +16,8 @@ class MailingContactImport(models.TransientModel):
     def action_import(self):
         """Import each lines of "contact_list" as a new contact."""
         self.ensure_one()
+        if not self.contact_list:
+            raise ValidationError(_("Please record contact list to import."))
```

Comment:
Vague message & no update in the `.pot` file added ;)

Reply:
could you suggest me a better message

Reply:
```suggestion
            raise ValidationError(_("Please add a contact list before importing it."))
```

or something like this :)

Reply:
Thanks, updated with the suggestion

Reply:
Now also add the term to the `.pot` file so it becomes available for Transifex to translate :)

Reply:
Thanks, updated .pot file